.github/workflows/release-connect-bump-versions.yml

```diff
@@ -0,0 +1,45 @@
+name: "[Release] connect bump versions"
+
+on:
+  workflow_dispatch:
+    inputs:
+      semver:
+        type: choice
+        description: semver
+        options:
+          - patch
+          - minor
+          - prerelease
+
+concurrency:
+  group: ${{ github.workflow }}-${{ github.head_ref || github.run_id }}
+  cancel-in-progress: true
+
+jobs:
+  bump-versions:
+    if: github.repository == 'trezor/trezor-suite'
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v4
+        with:
+          # Number of commits to fetch. 0 indicates all history for all branches and tags.
+          fetch-depth: 0
+          submodules: true
+
+      - name: Setup node
+        uses: actions/setup-node@v4
+        with:
+          node-version-file: ".nvmrc"
+
+      - name: Install dependencies
+        run: yarn install
+
+      - name: Set git for trezor-ci
+        run: |
+          git config --global user.name "trezor-ci"
+          git config --global user.email "${{ secrets.TREZOR_BOT_EMAIL }}"
+
+      - name: Check dependencies to update
+        run: |
+          node ./ci/scripts/connect-bump-versions.js ${{ github.event.inputs.semver }}
```

Comment:
It is good idea to use typescript, I will experiment with that. 
But anyway doing `yarn install` is still required since we use for example library `semver` in some of the helper functions used in this script.

Reply:
@mroz22 I make the scripts to use `tsx` I just added some simple typing for now but can keep typing it further.